### PR TITLE
CVQ2-225: Update content for empty segment

### PR DESCRIPTION
**Pull request by karlbaker02**
This PR updates the content that we show for an empty segment in the frontend. It also renames the page and endpoint from `no-mobile-phone` to `verify-will-not-work-for-you`.

Welsh translations are queued and should be updated in the frontend once available.

Solo: @karlbaker02

### CODE REVIEW

File: config/locales/cy.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ cy:
     other_identity_documents: dogfennau-hunaniaeth-eraill
     select_phone: dewis-ffon
     will_it_work_for_me: a-fydd-yn-gweithio-i-mi
-    no_mobile_phone: dim-ffon-symudol
+    verify_will_not_work_for_you: dim-ffon-symudol
```
Comment (jakubmiarka) [inline config/locales/cy.yml]: I would suggest putting the english version here, as the welsh is now 'incorrect' and makes it easier to spot the missing translation (if the welsh translations are delayed or done by other team)

Comment (karlbaker02) [reply config/locales/cy.yml]: Ah gotcha, so it would be `verify-will-not-work-for-you` for both en and cy until cy translations come through?

Comment (karlbaker02) [reply config/locales/cy.yml]: Would you suggest that we do this here?

Comment (jakubmiarka) [reply config/locales/cy.yml]: Given the current _low usage_ of this page, I think we can leave it, as long as we're expecting the translations soonish... ;)

Comment (karlbaker02) [reply config/locales/cy.yml]: Hopefully by 03/10.

**Pull request merged**